### ISSUE TITLE
Stencil

### ISSUE THREAD

**Issue by kipanshi** [gothinkster/realworld#325]
It would be nice to have the implementation build with StencilJS
https://stenciljs.com/

**Comment by hcavalieri** [comment]
This weekend I finally had the time and energy to start building something with Stencil, so I figured Conduit would be a good app to build... I'm ~80% done with an initial version, before community feedback and overall tweakings 😄 

For the source code, please refer to [this repo](https://github.com/hcavalieri/stencil-realworld-app). The current demo is [available here](https://stencil-realworld.netlify.com/).

I have a tiny TODO list in the `readme,md` file and some `TODO:` comments across the project, feel free to send a PR or file some issues (and don't go easy on me, it's my first Stencil project ever, throw all the feedback you have!)

Have a good one 🙌

**Comment by anishkny** [comment]
Great stuff! Quick note: the fonts seem off compared to https://angular.realworld.io/

<img width="1017" alt="screen shot 2019-01-19 at 6 41 47 pm" src="https://user-images.githubusercontent.com/357499/51433525-e9930f80-1c19-11e9-8a3f-1eeb22d99c3d.png">

**Comment by hcavalieri** [comment]
Good point, @anishkny !

I've finished styling the app and I'd say it's 95% done... What's left is handling errors for users to see and doing some testing to show how it's done in Stencil...

When you have the time, think you can go through the codebase and give me some feedback on what should be changed for it to be merged? Have a good one 😄

**Comment by hcavalieri** [comment]
@anishkny and @Cameron-C-Chapman you can already add Stencil's RealWorld app to the front-end list, the repo can be found here: https://github.com/hcavalieri/stencil-realworld-app

The demo is hosted at https://stencil-realworld.netlify.com/

**Comment by anishkny** [comment]
I tried it out locally and works great!! 🎉 

Couple of suggestions (not strictly required):

* Please add TravisCI/CircleCI/Azure Pipelines as it is really helpful
* Add badge to mark repo as RealWorld FE:

[![RealWorld Frontend](https://img.shields.io/badge/realworld-frontend-%23783578.svg)](http://realworld.io)
```
[![RealWorld Frontend](https://img.shields.io/badge/realworld-frontend-%23783578.svg)](http://realworld.io)
```

LGTM!

**Comment by hcavalieri** [comment]
Thanks for the quick reply, Anish!

The badge is added, I'll look into CI as soon as I have the time and energy, do agree it's super relevant for this project, plus it's going to be a good opportunity to learn more about it 😄

**Comment by anishkny** [comment]
Closed by #343 
Good job!
